File: Program's_Contributed_By_Contributors/JavaScript_Programs/factorial.js
// program to find the factorial of a number

// take input from the user
const number = parseInt(prompt("Enter a positive integer: "));

// checking if number is negative
if (number < 0) {
  console.log("Error! Factorial for negative number does not exist.");
}

// if number is 0
else if (number === 0) {
  console.log(`The factorial of ${number} is 1.`);
}

// if number is positive
else {
  let fact = 1;
  for (i = 1; i <= number; i++) {
    fact *= i;
  }
  console.log(`The factorial of ${number} is ${fact}.`);
}

File: Program's_Contributed_By_Contributors/C++/Factorial.cpp
#include<iostream>
using namespace std;

int fact(int n);

int main(){
    int n;
    cout<<"Enter the number to find factorial: ";
    cin>>n;
    cout<<fact(n);

    return 0;
}

int fact(int n){
    int ans=1;
    for(int i=n;i>0;i--){
        ans*=i;
    }
    return ans;
}

int fact(int n){
    if(n==1)
        return 1;
    else
        return n*fact(n-1);
}